app/cli/cmd/attestation_add.go

```diff
@@ -110,7 +110,7 @@ func newAttestationAddCmd() *cobra.Command {
 
 					return nil
 				},
-				backoff.NewExponentialBackOff(backoff.WithMaxElapsedTime(10*time.Second)),
+				backoff.NewExponentialBackOff(backoff.WithMaxElapsedTime(3*time.Minute)),
```

Comment:
the number is arbirtrary tbh, the default value is 60 minutes with I think it's too much